sdk-python/awaazde/utils.py

```diff
@@ -1,20 +1,21 @@
 import csv
+import logging
+import pandas as pd
 from datetime import datetime
-
 from constants import CommonConstants
 
 from . import AwaazDeAPI
 
 
-class ADMessageUtils(object):
+class ADUtils(object):
     def __init__(self, org, user, password):
         """
         Initialize Awaazde Authentication and Call Manager
         :return:
         """
         self.awaazde_api = AwaazDeAPI(org, user, password)
 
-    def get_messages(self, filters):
+    def get_created_messages(self, filters):
         """
             Gets all messages from awaazde API based on the filters passed
         """
```

Comment:
Please see [how this can be used](https://docs.python.org/3/library/urllib.parse.html) instead of implementing this by hand